### PR TITLE
feat: back-to-top

### LINKED ISSUE #27: Add Back to Top
Screenshot 2023-01-16 at 16 26 38

Clicking the above scrolls to the top of the page. It only appears when scrolling.

### PR DESCRIPTION
**Pull request by solaris007**
Add back-to-top feature. Delayed activation and only appears after scrolling a bit.

Fix #27

Test URLs:
- Before: https://main--elco-aerotop-sg--hlxsites.hlx.page/de/
- After: https://27-add-back-to-top--elco-aerotop-sg--hlxsites.hlx.page/de/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -293,7 +293,7 @@ main {
 }
 
 main .section {
-  padding-top: 0 30px 65px 30px;
+  padding: 0 30px 65px;
```
Comment (solaris007) [inline styles/styles.css:296]: this was needed to fix linting. this was checked in to main without lint check @cuzuco2

File: scripts/scripts.js [hunk 2]
```diff
@@ -111,6 +111,28 @@ export function addFavIcon(href) {
...
+  const link = document.createElement('a');
+  link.setAttribute('href', '#');
+  link.innerHTML = 'Go to Top';
+
+  backToTop.append(link);
+  main.append(backToTop);
+
+  document.addEventListener('scroll', (e) => {
+    // back to top only appears after a bit of scrolling
+    if (window.scrollY <= 200) {
```
Comment (dzehnder) [inline scripts/scripts.js:127]: might be different depending on device/monitor. Maybe we should use the viewport height instead

Comment (solaris007) [reply scripts/scripts.js:127]: the viewport height is constant. could you make an example on how to use it in this context?

Comment (dzehnder) [reply scripts/scripts.js:127]: `window.innerHeight` will return the height of the viewport. It is only constant to the currently visible viewport of the browser, which is different depending on the size

Comment (dzehnder) [reply scripts/scripts.js:127]: for example:
`if (window.scrollY <= window.innerHeight * 0.5) {}`
to only show it if the user scrolled half way through the window.
But I just checked on their website, it looks to me like they also hardcoded it. I guess we can keep it like that :)

**Review by dzehnder — APPROVED** [review]

**Pull request merged**